docs/reference/docs/reindex.asciidoc

```diff
@@ -141,8 +141,6 @@ If the request contains `wait_for_completion=false`, {es}
 performs some preflight checks, launches the request, and returns a
 <<tasks,`task`>> you can use to cancel or get the status of the task.
 {es} creates a record of this task as a document at `_tasks/<task_id>`.
-When you are done with a task, you should delete the task document so
-{es} can reclaim the space.
```

Comment:
Looks like the same sentence is present in `docs/reference/docs/update-by-query.asciidoc#L102-L103` should it be rmeoved as well?